tests/requests/test_views.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from django.urls import reverse
+from django_webtest import DjangoTestApp
+
+from vitrina.requests.factories import RequestFactory
+from vitrina.requests.models import Request
+
+
+@pytest.mark.django_db
+def test_request_create(csrf_exempt_django_app: DjangoTestApp):
+    resp = csrf_exempt_django_app.post(reverse("request-create"), {
+        'title': "Request",
+        'description': "Description"
+    })
```

Comment:
Please, always use webtest form handling functionality as documented here:

https://docs.pylonsproject.org/projects/webtest/en/latest/forms.html

This way you won't need `csrf_exempt_django_app` and more things will be tested , like if actual form and form fields exists in the page.